support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.jsx

```diff
@@ -278,13 +317,15 @@ const CardForm = (props: PropTypes) => {
 
   const handleCardSetupForRecurring = (clientSecret: string): void => {
     const cardElement = elements.getElement(CardNumberElement);
-    stripe.handleCardSetup(clientSecret, cardElement).then((result) => {
-      if (result.error) {
-        handleStripeError(result.error);
-      } else {
-        props.onPaymentAuthorised(result.setupIntent.payment_method);
-      }
-    });
+    stripe
+      .handleCardSetup(clientSecret, cardElement)
```

Comment:
I think we can actually support this field in the recurring flow too without having to involve Zuora etc by passing in ```
billing_details: {
          address: { postal_code: zipCode },
        },
```
as the third argument to `handleCardSetup`. 

See: https://stripe.com/docs/js/deprecated/handle_card_setup_element